crates/proof-of-sql/src/sql/ast/group_by_util.rs

```diff
@@ -183,3 +186,29 @@ pub(super) fn compare_indexes_by_columns<S: Scalar>(
         .find(|&ord| ord != Ordering::Equal)
         .unwrap_or(Ordering::Equal)
 }
+
+/// Compares the tuples (group_by[0][i], group_by[1][i], ...) and
+/// (group_by[0][j], group_by[1][j], ...) in lexicographic order.
+///
+/// Identical in functionality to [compare_indexes_by_columns]
+pub(super) fn compare_indexes_by_owned_columns<S: Scalar>(
```

Comment:
Not that we need this now but I wonder whether we should have a trait for common functionalities of different kinds of columns.